### ISSUE TITLE
Docker container fails to start since v3.3

### ISSUE THREAD

**Issue by lbr38** [bluewave-labs/Checkmate#3239]
**Describe the bug**
After an upgrade to v3.3, checkmate docker container fails to start.

**To Reproduce**
1. Upgrade to v3.3 using the provided docker image, with docker-compose (https://raw.githubusercontent.com/bluewave-labs/Checkmate/develop/docker/dist-mono/docker-compose.yaml)
2. Check container status with `docker ps`
3. See it fails to start

```
docker ps

CONTAINER ID   IMAGE                                                 COMMAND                  CREATED          STATUS                          PORTS           NAMES
56c3494a67bc   ghcr.io/bluewave-labs/checkmate-backend-mono:latest   "docker-entrypoint.s…"   10 minutes ago   Restarting (1) 24 seconds ago                   checkmate-server
```

```
docker logs checkmate-server

UPTIME_APP_API_BASE_URL=https://checkmate.xxxxxx.xxx/api/v1
UPTIME_APP_CLIENT_HOST=https://checkmate.xxxxxx.xxx
(node:28) [MONGOOSE] Warning: `errors` is a reserved schema pathname and may break some functionality. You are allowed to use it, but use at your own risk. To disable this warning pass `suppressReservedKeysWarning` as a schema option.
(Use `node --trace-warnings ...` to show where the warning was created)
2026-02-04T12:18:42.006Z error: [MongoDB](connect) BSON field 'create.timeseries' is an unknown field. (stack: [
  {
    "function": "Connection.sendCommand",
    "file": "/app/server/node_modules/mongodb/lib/cmap/connection.js",
    "line": 306,
    "column": 27
  },
  {
    "function": "process.processTicksAndRejections",
    "file": "node:internal/process/task_queues",
    "line": 95,
    "column": 5
  },
  {
    "function": "async Connection.command",
    "file": "/app/server/node_modules/mongodb/lib/cmap/connection.js",
    "line": 334,
    "column": 26
  },
  {
    "function": "async Server.command",
    "file": "/app/server/node_modules/mongodb/lib/sdam/server.js",
    "line": 194,
    "column": 29
  },
  {
    "function": "async tryOperation",
    "file": "/app/server/node_modules/mongodb/lib/operations/execute_operation.js",
    "line": 213,
    "column": 32
  },
  {
    "function": "async executeOperation",
    "file": "/app/server/node_modules/mongodb/lib/operations/execute_operation.js",
    "line": 78,
    "column": 16
  },
  {
    "function": "async createCollections",
    "file": "/app/server/node_modules/mongodb/lib/operations/create_collection.js",
    "line": 101,
    "column": 18
  },
  {
    "function": "async Db.createCollection",
    "file": "/app/server/node_modules/mongodb/lib/db.js",
    "line": 160,
    "column": 16
  },
  {
    "function": "async Function.createCollection",
    "file": "/app/server/node_modules/mongoose/lib/model.js",
    "line": 1242,
    "column": 7
  },
  {
    "function": "async Function.syncIndexes",
    "file": "/app/server/node_modules/mongoose/lib/model.js",
    "line": 1315,
    "column": 7
  }
])
```

**Expected behavior**
The container must be up an running

**Comment by ajhollid** [comment]
Hi @lbr38 ,

This looks like a MongoDB version issue; the project requires MongoDB >= 5.0 now.

If your database was created on a version < 5.0, then you will need to upgrade to a more recent version.  You can follow [these directions](https://www.mongodb.com/docs/manual/release-notes/8.0-upgrade-standalone/#std-label-8.0-upgrade-standalone) to upgrade.  

Note I don't think you can leapfrog versions, so you have to do 4 -> 5 -> 6 -> 7 -> 8 if you want to get to the latest version.

**Comment by lbr38** [comment]
Unfortunately I ran with an error with mongodb > 5.0 on my CPU, this is why I was running mongo 4.4.18:
```
WARNING: MongoDB 5.0+ requires a CPU with AVX support, and your current system does not appear to have that!
```

I am afraid I must now say goodbye to Checkmate 🥲​

Thanks for your help anyway!

**Comment by ajhollid** [comment]
Sorry to hear that @lbr38 !  

You could always keep running the older version of Checkmate that doesn't require >= 5.0 for the time being if you like.

Keep your eyes open in the future as well, we've laid the groundwork for swapping out MongoDB for something else, so as soon as someone has time to write repository classes for some other database, we can support that.
